packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -0,0 +1,715 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
+import Meeting from '@webex/plugin-meetings/src/meeting';
+import Members from '@webex/plugin-meetings/src/members';
+import Roap from '@webex/plugin-meetings/src/roap';
+import StateMachine from 'javascript-state-machine';
+import MeetingRequest from '@webex/plugin-meetings/src/meeting/request';
+import LocusInfo from '@webex/plugin-meetings/src/locus-info';
+import MediaProperties from '@webex/plugin-meetings/src/media/properties';
+import Events from '@webex/plugin-meetings/src/events';
+import WebRTCStats from '@webex/plugin-meetings/src/stats';
+import MeetingUtil from '@webex/plugin-meetings/src/meeting/util';
+import Media from '@webex/plugin-meetings/src/media/index';
+import PeerConnectionManager from '@webex/plugin-meetings/src/peer-connection-manager';
+import ReconnectionManager from '@webex/plugin-meetings/src/reconnection-manager';
+
+describe('plugin-meetings', () => {
+  describe('meeting index', () => {
+    let meeting;
+    const mercury = {};
+    const log = sinon.stub().returns(true);
+    const trigger = sinon.stub().returns(true);
+    const spark = {
+      trigger,
+      internal: {mercury},
+      logger: {log},
+      meetings: {config: {mediaSettings: {}}}
+    };
+    beforeEach(() => {
+      meeting = new Meeting(
+        {
+          userId: 'test',
+          resource: 'test',
+          deviceUrl: 'test',
+          locus: {url: 'test'}
```

Comment:
I think it's helpful to put realistic looking data in tests. Otherwise you get a string of `test` everywhere and it's possible it's not the value you think it is but some other prop.